en/docs/security-announcements/cve-justifications/2025/GHSA-593m-55hh-j8gv.md

```diff
@@ -0,0 +1,46 @@
+---
```

Comment:
Please update the https://github.com/wso2/docs-security/blob/main/en/mkdocs.yml file as well.